### PR TITLE
Describe using non-package DESCRIPTION files in `use_description()` docs

### LINKED ISSUE #979: Make docs more clear that use_description() works in a non-package project
So I recently learned that one can include a [DESCRIPTION file outside of packages](https://twitter.com/bradleyboehmke/status/1215703648384225281), it's something I perceive as good practice.

image

However, to the best of my knowledge, usethis does not currently support that.

```r
r$> usethis::use_description()                                                                                
Error: Path '/my/current/directory/' does not appear to be inside a project or package.
In addition: Warning message:
In path_file(base_path) : restarting interrupted promise evaluation
```

### PR DESCRIPTION
**Pull request by malcolmbarrett**
Closes #979

### CODE REVIEW

File: R/description.R [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
-#' fields:
-#' * `fields` argument of [create_package()] or [use_description()].
-#' * `getOption("usethis.description")` or `getOption("devtools.desc")`. The
+#' fields: * `fields` argument of [create_package()] or [use_description()]. *
+#' `getOption("usethis.description")` or `getOption("devtools.desc")`. The
 #' devtools option is consulted only for backwards compatibility and it's
-#' recommended to switch to an option named "usethis.description".
-#' * Defaults built into usethis.
+#' recommended to switch to an option named "usethis.description". * Defaults
+#' built into usethis.
```
Comment (jennybc) [inline R/description.R]: This looks like a mangled bullet list. I suspect it was partially mangled to start with. This happens when I reflow roxygen comments. Can you straighten it out?

File: R/description.R [hunk 2]
```diff
@@ -16,24 +23,18 @@
...
-#'                           comment = c(ORCID = "YOUR-ORCID-ID"))',
-#'     License = "MIT + file LICENSE",
-#'     Language =  "es"
-#'   )
-#' )
-#' ```
+#' ``` options( usethis.description = list( `Authors@R` = 'person("Jane", "Doe",
+#' email = "jane@example.com", role = c("aut", "cre"), comment = c(ORCID =
+#' "YOUR-ORCID-ID"))', License = "MIT + file LICENSE", Language =  "es" ) ) ```
 #'
```
Comment (jennybc) [inline R/description.R]: This got mangled. Pretty sure there should be no change here.

File: R/description.R (hunk 1 above)
Comment (malcolmbarrett) [reply R/description.R]: ack, will do!

**Comment by malcolmbarrett** [comment]
Ok, the diffs look much cleaner now

**Comment by jennybc** [comment]
Thanks!

**Pull request merged**